libbeat/management/config.go

```diff
@@ -0,0 +1,72 @@
+package management
+
+import (
+	"os"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/file"
+	"github.com/elastic/beats/libbeat/paths"
+	"github.com/elastic/beats/libbeat/setup/kibana"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// Config for central management
+type Config struct {
+	// true when enrolled
+	Enabled bool
+
+	// TODO use beat.Keystore() for access_token
+	AccessToken string
+
+	Kibana *kibana.Config
+
+	Configs []struct {
+		Name   string
+		Config *common.Config
+	}
+}
+
+// Load settings from its source file
+func (c *Config) Load() error {
+	path := paths.Resolve(paths.Data, "management.yaml")
```

Comment:
I applaud your attempt to use the [official](http://yaml.org/faq.html) YAML file extension, but we seem to use `.yml` for everything.

Reply:
🙈